runners/direct-java/src/main/java/org/apache/beam/runners/direct/BoundedReadEvaluatorFactory.java

```diff
@@ -73,7 +73,7 @@ public void cleanup() {}
    * already done so.
    */
   private <OutputT> TransformEvaluator<?> getTransformEvaluator(
-      final AppliedPTransform<?, PCollection<OutputT>, Bounded<OutputT>> transform) {
```

Comment:
I think so, still investigating that.  ejc generated class files don't pass findbugs and the annotation generated classes are in a different location which confuses checkstyle.   If you apply this diff to master and run with:

`mvn -Dmaven.test.skip.exec=true -Dcheckstyle.skip=true -Dfindbugs.skip=true clean install`

then the error should pop up.  Just need to figure out how to put this in a profile or similar.

`diff --git a/pom.xml b/pom.xml
index f9e0479..edb5951 100644
--- a/pom.xml
+++ b/pom.xml
@@ -710,13 +710,13 @@

         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.1</version>
+          <version>3.3</version>
           <configuration>
             <source>1.7</source>
             <target>1.7</target>
             <compilerArgs>
               <arg>-Xlint:all</arg>
-              <arg>-Werror</arg>
+              <!--arg>-Werror</arg-->
               <!-- Override options warnings to support cross-compilation -->
               <arg>-Xlint:-options</arg>
               <!-- Temporary lint overrides, to be removed over time. -->
@@ -735,7 +735,15 @@
             <showWarnings>true</showWarnings>
             <!-- Another temp override, to be set to true in due course. -->
             <showDeprecation>false</showDeprecation>
+            <compilerId>jdt</compilerId>
           </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.eclipse.tycho</groupId>
+                        <artifactId>tycho-compiler-jdt</artifactId>
+                        <version>0.26.0</version>
+                    </dependency>
+               </dependencies>
         </plugin>

         <plugin>
`